src/osp/core/array_view.h

```diff
@@ -36,34 +38,52 @@ namespace osp
 using Corrade::Containers::ArrayView;
 using Corrade::Containers::arrayView;
 
+/**
+ * @brief Wraps a Corrade ArrayView or StridedArrayView to use as a 2D array of equally sized rows
+ */
 template<typename T>
 struct ArrayView2DWrapper
```

Comment:
Since this class is only supposed to be used with ArrayView or StridedArrayView, you might want to consider adding template specializations for only those two, and leaving the "default" template declaration of the struct undefined.

The lazy way of just leaving it like this is also fine, of course.

Reply:
Or a requires clause.

either way.

Reply:
I guess you mean a concept for array views?

I'll be lazy on this as there's already a short comment explaining what this type is, and it's a little too simple.